.github/workflows/_python-tests.yml

```diff
@@ -92,6 +92,7 @@ jobs:
         test-globs: ["chromadb/test/db/test_system.py",
                    "chromadb/test/property/test_collections.py",
                    "chromadb/test/property/test_add.py",
+                   "chromadb/test/property/test_filtering.py",
```

Comment:
Is it ok to turn it on before I verify that it works?

Reply:
I don't intend to merge this, was just putting it up for you to fork/copy/use